### PR TITLE
Changes to the use of arm_64bit

### PR DESCRIPTION
**Pull request by JamesH65**

### CODE REVIEW

File: configuration/config-txt/boot.md [hunk 1]
```diff
@@ -35,7 +35,13 @@ Set the `disable_commandline_tags` command to `1` to stop `start.elf` from filli
 
 ## kernel
 
-`kernel` is the alternative filename on the boot partition to use when loading the kernel. The default value on the Pi 1, Pi Zero, and Compute Module is `kernel.img`, and on the Pi 2, Pi 3, and Compute Module 3 it is `kernel7.img`. If `kernel8.img` is present on the Pi 3 or Compute Module 3, it will be loaded in preference and entered in 64-bit mode. **NOTE**: This must be an uncompressed kernel image file.
+`kernel` is the alternative filename on the boot partition to use when loading the kernel. The default value on the Pi 1, Pi Zero, and Compute Module is `kernel.img`, and on the Pi 2, Pi 3, and Compute Module 3 it is `kernel7.img`. On the Pi4, it is `kernel7l.img`. **NOTE**: This must be an uncompressed kernel image file.
```
Comment (pelwell) [inline configuration/config-txt/boot.md]: The "uncompressed kernel image" requirement applies to 64-bit kernels.

File: configuration/config-txt/misc.md [hunk 2]
```diff
@@ -11,14 +11,6 @@ The [warning symbols](../warning-icons.md) can be disabled using this option, al
 
 Sets the Videocore logging level. The value is a Videocore-specific bitmask.
 
-## arm_control
```
Comment (pelwell) [inline configuration/config-txt/misc.md]: I think it's worth copying the deprecation notice to boot.md.

File: configuration/config-txt/boot.md (hunk 1 above)
Comment (lurch) [reply configuration/config-txt/boot.md]: But 32 bit kernels can be compressed or uncompressed? Is it always still called `kernel.img` regardless of whether it's compressed or not? (And is it worth mentioning what kernel compression schemes are supported?)
:man_shrugging:

File: configuration/config-txt/boot.md [hunk 3]
```diff
@@ -35,11 +35,11 @@ Set the `disable_commandline_tags` command to `1` to stop `start.elf` from filli
...
+`kernel` is the alternative filename on the boot partition to use when loading the kernel. The default value on the Pi 1, Pi Zero, and Compute Module is `kernel.img`, and on the Pi 2, Pi 3, and Compute Module 3 it is `kernel7.img`. On the Pi4, it is `kernel7l.img`. 
 
 ## arm_64bit
 
-If set to non-zero, forces the kernel loading system to assume a 64-bit kernel, starts the processors up in 64bit mode, and sets `kernel8.img` to be the kernel image loaded, unless there is an explicit `kernel` option defined in which case that is used instead. Defaults to 0 on all platforms. **NOTE**: Kernel images, both 32 and 64bit, must be uncompressed image files.
+If set to non-zero, forces the kernel loading system to assume a 64-bit kernel, starts the processors up in 64bit mode, and sets `kernel8.img` to be the kernel image loaded, unless there is an explicit `kernel` option defined in which case that is used instead. Defaults to 0 on all platforms. **NOTE**: 64bit kernels must be uncompressed image files.
```
Comment (pelwell) [inline configuration/config-txt/boot.md]: Just standardise on "64-bit" (as opposed to "64bit") and we're good to go.

**Comment by pelwell** [comment]
Looks good to me.

**Comment by JamesH65** [comment]
Cannot wait for CE, so will merge this straight away.

**Pull request merged**